airflow/settings.py

```diff
@@ -256,6 +256,16 @@ def __init__(self):
     def remove(*args, **kwargs):
         pass
 
+    def get_bind(
```

Comment:
The failing tests should be marked as @pytest.mark.db_test instead. Not sure which tests failed with it and why (somewhat the exception above would have to be ignored), but This session should have never been initialized in the "non-db" tests cases. See https://github.com/apache/airflow/blob/main/contributing-docs/testing/unit_tests.rst#db-and-non-db-tests for more explanation

Reply:
Will see if they fail again, don't remember which tests started failling...